cypress/integration/scheduler.ts

```diff
@@ -36,61 +36,101 @@ context('Scheduler tests', () => {
     cy.finishedLoading();
 
     cy.contains('Upcoming beam times').should('not.exist');
+    cy.logout();
```

Comment:
For some weird reasons the user stays logged in, couldn't figure out why, so I added an explicit logout

Reply:
Maybe it has to do something with the cypress upgrade or it is not because of that what do you think?

Reply:
I think it's related to the request we make to change the booking status, either a bug or something else.